boot/seal.go

```diff
@@ -101,6 +103,10 @@ func sealKeyToModeenv(key secboot.EncryptionKey, model *asserts.Model, modeenv *
 		return fmt.Errorf("cannot seal the encryption key: %v", err)
 	}
 
+	if err := stampSealedKeys(InstallHostWritableDir); err != nil {
```

Comment:
Eventually this will need to use a more generic *Dir variable right? Or does it just work out that this same mount point exists in run mode as well as install mode?

Reply:
well, seal install is only used at install so this is right

Reply:
ah right sorry I missed that this was in sealKeyToModeenv, makes sense